### PR TITLE
Update view_only.md

### PR DESCRIPTION
**Pull request by ibartana**
Added usage of view-only wallet

### CODE REVIEW

File: knowledge-base/user-guides/view_only.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: static_page
 title: "View Only Wallets"
-title-pre-kick: "Creating a view only wallet "
+title-pre-kick: "Creating aand using a view only wallet "
```
Comment (fluffypony) [inline knowledge-base/user-guides/view_only.md]: "aand" should be "and"

File: knowledge-base/user-guides/view_only.md [hunk 2]
```diff
@@ -12,12 +12,25 @@ attribution: "<!-- Icon is based on work by Freepik (http://www.freepik.com) and
...
+We'll call the new wallet is ViewWallet.
+* Finally, move the three Viewwallet files (viewWallet, viewWallet.address and viewWallet.KEYS) to an online computer.
 
-Next, create your view-only wallet by typing `monero-wallet-cli --generate-from-view-key wallet-name`. The last argument will be your new wallet's file name. You will be prompted for `Standard address` and `View key` by the wallet. Paste in your original wallet's address and secret view key. Next, enter and confirm a password for your new wallet and you're done.
+### Creating, signing and broadcasting transctions using ViewWallet and OfflineWallet
+
+To create a transaction from the viewWallet, start with the ViewWallet: 
+* Run monero-wallet-cli on the online computer and load the ViewWallet.
+* Next, use the monero-wallet-cli command `transfer` with parameters ADDRESS and AMOUNT. The viewWallet will create a new file in its directory called `unsigned-monero-transaction`. 
+* Copy that file to an SD card/USB key and move it to the directory of the OfflineWallet. For extra protection, use an SD card for that purpose only.
```
Comment (jonathancross) [inline knowledge-base/user-guides/view_only.md]: > For extra protection, use an SD card for that purpose only.

Seems you are suggesting that they should not use the SD card or USB key for any other purpose?

File: knowledge-base/user-guides/view_only.md [hunk 3]
```diff
@@ -12,12 +12,25 @@ attribution: "<!-- Icon is based on work by Freepik (http://www.freepik.com) and
...
+* Next, using monero-wallet-cli load the OfflineWallet, and use the command `sign_transfer`. It will replace the file 	`unsigned-monero-transaction` with a file `signed-monero-transaction`.
```
Comment (jonathancross) [inline knowledge-base/user-guides/view_only.md]: Nit: please remove extra blank spaces here and below.

**Comment by jonathancross** [comment]
Looks like this information has been moved to a different file as part of the website redesign:

[/resources/user-guides/view_only.md](https://github.com/monero-project/monero-site/blob/master/resources/user-guides/view_only.md)

**Comment by rehrar** [comment]
Hey @ibartana thanks for your contribution. Unfortunately this PR needs a rebase. It's really good stuff, and we would love for you to be able to make the changes so we can merge it.

Let me know if you have any questions. You can see @jonathancross comment to see what needs to change.

**Comment by ibartana** [comment]
@jonathancross - I've almost done writing a massive, 18 page step by step guide titled "The complete guide to creating and using a high value Monero cold storage wallet" that is a lot more thorough and comprehensive. It's currently a PDF, but I can make it into html based document if needed. Unfortunately, I'm a github noob - how do I store this as a visible Monero guide on the site? 

Lacking that - how do I rebase what I've added above?

**Comment by rehrar** [comment]
@ibartana if you check the readme of this repo you'll see I've updated it with a comprehensive list of how to do a lot of things on the website. One of them is how to make a User Guide. Let me know if the instructions are confusing. :)

**Comment by jonathancross** [comment]
Thanks @rehrar.

@ibartana Yes, git + GitHub can be a bit difficult at first.  [Here is a Cheat Sheet I created](/jonathancross/jc-docs/blob/master/Git-CheatSheet.md) which might be of use.

At this point you might want to consider making a backup of your changes somewhere, closing this pull request, then starting over using the [information suggested by rehar](https://github.com/monero-project/monero-site#50-how-to-make-a-user-guide).

**Comment by fluffypony** [comment]
This is still pending changes. If the changes aren't made within the next 30 days we'll have to close this PR.

**Comment by rehrar** [comment]
Any update @ibartana ?

**Comment by fluffypony** [comment]
Closed as abandoned

**Pull request closed without merging**